modin/pandas/base.py

```diff
@@ -3710,25 +3845,25 @@ def sem(self, _method="sem", *args, **kwargs):
         )
 
     def std(self, ddof=1, *args, **kwargs):
-        if self.resample_args[1] == 0 or self.resample_args[1] == "index":
-            result = self.__groups.std(ddof=ddof, *args, **kwargs)
-        else:
-            result = self.__groups.std(ddof=ddof, *args, **kwargs).T
-        return result
+        return self._dataframe.__constructor__(
+            query_compiler=self._query_compiler.resample_std(
+                self.resample_args, *args, ddof=ddof, **kwargs
+            )
+        )
 
     def sum(self, _method="sum", min_count=0, *args, **kwargs):
-        if self.resample_args[1] == 0 or self.resample_args[1] == "index":
+        if self.resample_args[1] == 0:
             result = self.__groups.sum(min_count=min_count, *args, **kwargs)
         else:
             result = self.__groups.sum(min_count=min_count, *args, **kwargs).T
         return result
 
     def var(self, ddof=1, *args, **kwargs):
-        if self.resample_args[1] == 0 or self.resample_args[1] == "index":
-            result = self.__groups.var(ddof=ddof, *args, **kwargs)
-        else:
-            result = self.__groups.var(ddof=ddof, *args, **kwargs).T
```

Comment:
is `__groups` field used somewhere after your changes?

Reply:
Yes, this is used in `get_group`, `prod` and `sum`